students/17th/hkk/hyeonsookang/user/views.py

```diff
@@ -0,0 +1,39 @@
+import json
+
+from django.http  import JsonResponse
+from django.views import View
+
+from user.models  import User
+
+class UserView(View):
+    def post(self, request):
+        try:
+            data             = json.loads(request.body)
+            passwordvalidity = len(data['password'])
+
+            if '@' not in data['email'] or '.' not in data['email']:
+                return JsonResponse({'MESSAGE': 'INVALID__EMAIL'}, status=401)
+
+            if User.objects.filter(email=data['email']).exists():
+                return JsonResponse({'MESSAGE': 'ALREADY_SIGNUP'}, status=409)
+
+            if User.objects.filter(email=data['phone_number']).exists():
+                return JsonResponse({'MESSAGE': 'ALREADY_SIGNUP'}, status=409)
+
+            if User.objects.filter(email=data['account']).exists():
+                return JsonResponse({'MESSAGE': 'ALREADY_SIGNUP'}, status=409)
+
+            if passwordvalidity < 8:
```

Comment:
```suggestion
            if passwordvalidity < PASSWORD_MINIMUM_LENGTH:
```
오히려 위와 같이 비교값 8을 상수로 저장해서 사용해주세요.